kivy/_event.pyx

```diff
@@ -840,25 +847,46 @@ cdef class EventDispatcher(ObjectWithUid):
             True
         '''
         cdef Property prop
-        if value is None:  # shortcut
-            prop = ObjectProperty(None, *largs, **kwargs)
-        if isinstance(value, bool):
-            prop = BooleanProperty(value, *largs, **kwargs)
-        elif isinstance(value, (int, float)):
-            prop = NumericProperty(value, *largs, **kwargs)
-        elif isinstance(value, string_types):
-            prop = StringProperty(value, *largs, **kwargs)
-        elif isinstance(value, (list, tuple)):
-            prop = ListProperty(value, *largs, **kwargs)
-        elif isinstance(value, dict):
-            prop = DictProperty(value, *largs, **kwargs)
+
+        if default_value:
```

Comment:
DRY - replacing the code within this if-block with the following code line:
```suggestion
        if default_value:
            largs = (value, ) + largs
```
... you could indent out the code in the `else:` code block (using it for both cases: default_value == True and == False).

Reply:
I've simplified it in an alternative way.

Reply:
great job - your simplification is more clean :)